trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk

```diff
@@ -158,18 +160,39 @@
   {% if paymentDetails.length === 0 %}
     <p class="govuk-body">Payment details will be displayed when payments have been entered on the premium payments tab.</p>
   {% else %}
-    {{ govukButton({
-      text: "Show filter",
-      classes: "govuk-button govuk-button--secondary govuk-!-margin-left-2",
-      id: "payment-details-show-filter-button",
-      attributes: {'data-cy': 'payment-details-show-filter-button'}
-    }) }}
+    <div class="moj-filter-layout">
 
-    {{ paymentDetailsTable.render({
-      reportId: reportId,
-      paymentDetails: paymentDetails,
-      userCanEdit: userCanEdit
-    }) }}
+      {{ paymentDetailsFiltersPanel.render({
+        filters: filters,
+        selectedFilters: selectedFilters,
+        keyword: keyword,
+        csrfToken: csrfToken
```

Comment:
where does this get rendered to be passed back? Is it part of the MoJ implementation or should I be expecting a csrfToken to be passed back in a form somewhere? (Sorry if I'm missing something obvious!)

Reply:
oops I forgot to update these, sorted it now thanks - removed the CSRF token as it isn't needed/used (at least for now)